### PR TITLE
Connect to the contracts

### PR DESCRIPTION
**Pull request by gabrielpoca**
With this change, the backend connects to the real contracts running on ropsten.

Extra:
* fixed jobs that absolutely have to run on boot.

### CODE REVIEW

File: src/config/index.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import assert from 'assert';
+
+export default {
+  ropsten: () => {
+    assert(process.env.ROPSTEN_RPC_URL);
+    assert(process.env.TESTNET_MNEMONIC);
+
+    return {
+      vault: '0x6e1d164c682698857EB22fB9a184B245329F7581',
```
Comment (0xrin1) [inline src/config/index.ts:9]: do we want this hardcoded?

**Review by 0xrin1 — COMMENTED** [review]
just that one comment tbh the rest LGTM

I really like the way the collect performance logic is much more straight forward now

File: src/config/index.ts (hunk 1 above)
Comment (gabrielpoca) [reply src/config/index.ts:9]: I guess? Do you think it's worth making an env variable? It's not like it's going to change

Comment (0xrin1) [reply src/config/index.ts:9]: I think there's no harm to using an .env variable to be honest, if it does need to change we don't need to make another commit to handle the change

we might still need to change something and then redeploy to testnet

Comment (gabrielpoca) [reply src/config/index.ts]: donest 👍

**Review by 0xrin1 — APPROVED** [review]

**Review by joaop21 — APPROVED** [review]
🚀

**Pull request merged**